packages/invoices/src/InboundShipment/api/hooks/document/useInboundDelete.ts

```diff
@@ -0,0 +1,47 @@
+import {
```

Comment:
Do you know how to get this to show up in the diff as a renamed (moved) file rather than a deleted file and a whole new file?